<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDlg</class>
 <widget class="QDialog" name="StartDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Защищенная печать</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="markPaperBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Учет листов</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="printOnMarkPaperBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Печать на учтеных листах</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="printWithAutoMarkPaperBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Печать документа с автоматическим учетом листов</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QComboBox" name="printerCBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Список разрешенных принтеров...</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="logList">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
